mesa_geo/geospace.py

```diff
@@ -103,13 +103,13 @@ def get_neighbors_within_distance(
         set center=True to calculate distance from center.
         """
         if center:
-            shape = agent.shape.center().buffer(distance)
+            shape = agent.geometry.center().buffer(distance)
```

Comment:
Should be `geometry = ...`? The argument in the `_get_rtree_intersections` function should be called `geometry` as well, because in L64, it is called with `possible_agents = self._get_rtree_intersections(agent.geometry)`?

Reply:
Thanks for spotting these!